modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/elements/base-protect-building-blocks.js

```diff
@@ -0,0 +1,14 @@
+export class BaseProtectBuildingBlocks extends $e.modules.hookData.Dependency {
```

Comment:
Can it be done by not calling the commands? (instead of hooking, something like a new element prop: `locked`)

Reply:
I'm not sure about it, please take a look here https://github.com/elementor/elementor/pull/18912/files#diff-c8985fb84145e8a65b8643b8c904f218a7d5c43a9528441d18d8cda782b52389R16 
- There are scenarios which it should be unlocked, please give me your suggestion.